src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
delegate most of the logic off to private methods - reading file, generating report, writing report to a file, and then just call those methods here, in the public method